### PR TITLE
Kodi update to 19.09 Beta1

### PR DESCRIPTION
**Pull request by CvH**
- updated Kodi and rebased patches
- updated ffmpeg to kodi 19.01b1
- updated binary/retro addons
- bumped addon version, hopefully the last api breakage at K19

### CODE REVIEW

File: packages/mediacenter/kodi/package.mk [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2017-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="kodi"
-PKG_VERSION="bb0699db41d50a613de88f2bbe8124d4edf269d3"
-PKG_SHA256="28bd28a85e47995d4965ac402228374662bb85ba0644c94c52f371d97c87db1b"
+PKG_VERSION="88e186e4b9d8f4d5a16af89a68071c0710f119bf"
```
Comment (HiassofT) [inline packages/mediacenter/kodi/package.mk:6]: please use the 19.0b1-Matrix tag instead of the githash

Comment (CvH) [reply packages/mediacenter/kodi/package.mk]: fixed

**Comment by CvH** [comment]
builds and runs for generic

**Comment by jernejsk** [comment]
quick test on AW R40 and H6, works ok

**Comment by HiassofT** [comment]
quick tests on RPi2 and RPi4 look good, too!

**Review by HiassofT — APPROVED** [review]

**Pull request merged**